Node/avg.js

```diff
@@ -0,0 +1,25 @@
+function calcAvgNum(num1, num2, num3){
+    return (num1 + num2 + num3) / 3
+}
+const args = process.argv.slice(2);
+if (args.length < 3){
```

Comment:
Avoid hard-coded number of arguments. Rather than expecting exactly 3 arguments, make the code flexible enough to handle any number of arguments.